### PR TITLE
init context, alternatives

### PR DESCRIPTION
**Pull request by marcusellison**

### CODE REVIEW

File: bdr/006-what-to-name-comparing-lists-scope.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# What to name comparing lists scope
+
+We need a name for the way we connect updates to one another
```
Comment (xh3b4sd) [inline bdr/006-what-to-name-comparing-lists-scope.md:3]: "Connecting" sounds more like what the internet calls "link". What I miss in our understanding is the place or box where updates are grouped together. Important here is the aspect of navigation. How do people find that place depending on what they want to do and how do they move between places.

Comment (xh3b4sd) [inline bdr/006-what-to-name-comparing-lists-scope.md:3]: "Connecting" sounds more like what the internet calls "link". What I miss in our understanding is the place or box where updates are grouped together. Important here is the aspect of navigation. How do people find that place depending on what they want to do and how do they move between places.

File: bdr/006-what-to-name-comparing-lists-scope.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Context
+
+When a user creates an update they generate a list of hashtags. However, in order to compare this list from one user to another we need to translate into fundamental units that better capture our context along similar dimensions. What should we call these units?
+
+### Alternatives
+
+- atoms
```
Comment (xh3b4sd) [inline bdr/006-what-to-name-comparing-lists-scope.md:11]: Atoms does not fit well. An atom is one of the smallest unit. What I understand here is to find a name for the collection of smallest units.

**Review by xh3b4sd — COMMENTED** [review]
Reading through it we are not on the same page with this one. Let's chat about it later in person.

File: bdr/006-what-to-name-comparing-lists-scope.md (hunk 1 above)
Comment (marcusellison) [reply bdr/006-what-to-name-comparing-lists-scope.md:3]: I understand. updating soon.

File: bdr/006-what-to-name-comparing-lists-scope.md (hunk 2 above)
Comment (marcusellison) [reply bdr/006-what-to-name-comparing-lists-scope.md:11]: updating context and alternatives.

**Comment by xh3b4sd** [comment]
Outdated, closing. Feel free to reopen.

**Pull request closed without merging**